app/webpacker/css/admin_v3/components/navigation.scss

```diff
@@ -61,6 +61,8 @@ nav.menu {
 }
 
 #admin-menu {
+  box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.05), 0px 2px 2px rgba(0, 0, 0, 0.07);
```

Comment:
Do you think we could keep borders in variables.scss? This would help us keep them consistent where possible.